open-banking-accelerator/components/com.wso2.openbanking.accelerator.identity/src/main/java/com/wso2/openbanking/accelerator/identity/app2app/model/DeviceVerificationTokenConstants.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.openbanking.accelerator.identity.app2app.model;
+
+/**
+ * Constants for DeviceVerificationToken.
+ */
+public class DeviceVerificationTokenConstants {
+
+    public static final String EXPIRY_TIME = "exp";
+    public static final String NOT_VALID_BEFORE = "nbf";
+    public static final String LOGIN_HINT = "loginHint";
```

Comment:
shall we consider snake case here like in CIBA? `login_hint`

Reply:
Resolved in [260e508](https://github.com/wso2/financial-open-banking/pull/67/commits/260e508da50ed60b02dc3fd2f2d27107b59691d2).